packages/first-boot-wizard/files/usr/lib/lua/firstbootwizard.lua

```diff
@@ -328,8 +328,8 @@ function apply_user_configs(configs)
     -- nixio.syslog("crit", "FBW apply_user_configs ssid "..ssid)
     uci_cursor:set("lime-defaults", 'wifi', 'ap_ssid', name)
     uci_cursor:set("lime-defaults", 'wifi', 'apname_ssid', name..'/%H')
-    uci_cursor:set("lime-defaults", 'wifi', 'adhoc_ssid', 'LiMe.'..name..'/%H')
-    uci_cursor:set("lime-defaults", 'wifi', 'ieee80211s_mesh_id', 'LiMe.'..name..'/%H')
+    uci_cursor:set("lime-defaults", 'wifi', 'adhoc_ssid', name..'.%H')
+    uci_cursor:set("lime-defaults", 'wifi', 'ieee80211s_mesh_id', name)
```

Comment:
this should be independent for the network name and constast accross all LibreMesh devices of al networks

Reply:
https://github.com/libremesh/lime-packages/pull/493/commits/3f50ca16dda23b02831cd2eec23cfa6e16b7e1a7